packages/scroll-points/__tests__/playwright/playwright.spec.js

```diff
@@ -13,6 +13,7 @@ test.describe('Scroll points > Functionality', { tag: '@all'}, () => {
 
 	test('Should have at least one activated scroll point by the time the page has scrolled to the bottom', async ({ page }) => {	
 		await page.evaluate(() => window.scrollTo(0, document.body.scrollHeight || document.documentElement.scrollHeight));
+		await page.waitForTimeout(5000);
```

Comment:
Running locally on a MacBook the test passes every time I've tried but nx flags it as 'flaky' because of this timeout. I can live with that if it's unavoidable to get this to pass on Safari in a GitHub Action.